build.sbt

```diff
@@ -197,13 +197,14 @@ lazy val zincCompile = (project in file("zinc-compile"))
   )
   .configure(addSbtUtilTracking)
 
-// Persists the incremental data structures using SBinary
+// Persists the incremental data structures using Protobuf
 lazy val zincPersist = (project in internalPath / "zinc-persist")
   .dependsOn(zincCore, zincCore % "test->test")
   .configure(addBaseSettingsAndTestDeps)
   .settings(
     name := "zinc Persist",
-    libraryDependencies += sbinary
+    compileOrder := sbt.CompileOrder.Mixed,
```

Comment:
We usually try to isolate the Java interfaces into subprojects like xsbti. If the API layer is depending on Scala, that's a smell.